README.md

```diff
@@ -19,13 +19,17 @@ The cluster definition file enables you to customize your Docker enabled cluster
 
 ## User guides
 
-* [ACS Engine](docs/acsengine.md) - shows you how to build and use the ACS engine to generate custom Docker enabled container clusters
+These guides show how to create your first deployment for each orchestrator:
+* [ACS Engine](docs/acsengine.md) - shows you how to build and use the ACS engine to generate custom container clusters
+* [DC/OS Walkthrough](docs/dcos.md) - shows how to create a DC/OS cluster on Azure
+* [Kubernetes Walkthrough](docs/kubernetes.md) - shows how to create a Kubernetes cluster on Azure
```

Comment:
I think some docs in [Kubernetes Walkthrough](docs/kubernetes.md) applies to both Linux and Windows, maybe it'd make more sense to add the link for Windows inside that page instead so that the readme links to the general Kubernetes docs and then the user can go to instructions for windows from there.